package.json

```diff
@@ -28,10 +28,12 @@
     "@types/react-native": "^0.71.0",
     "conventional-changelog-conventionalcommits": "^5.0.0",
     "eslint": "^8.33.0",
+    "i18next": "^22.4.9",
     "jest": "^29.4.1",
     "metro-react-native-babel-preset": "^0.74.1",
     "prettier": "^2.8.3",
     "react": "^18.2.0",
+    "react-i18next": "^12.1.5",
```

Comment:
All peerDependencies also need to be added as devDependencies in order to work locally.